scripts/types.ts

```diff
@@ -7,6 +7,7 @@ export const protocols = [
   'velodrome',
   'fonbnk',
   'aave',
+  'celo-transactions',
```

Comment:
open to naming suggestions

Reply:
sounds good -- could we also make, like, an alias of `celo` called `celo-transactions` for calling fetchReferrals?

Reply:
yes good idea, I can make this with the fetch referrals update